src/app/content/components/Toolbar/index.tsx

```diff
@@ -3,6 +3,8 @@ import flow from 'lodash/fp/flow';
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { connect } from 'react-redux';
+import { featureFlagsEnabled as featureFlagsEnabledSelector } from '../../../featureFlags/selectors';
```

Comment:
i like to treat selectors like verbs, often i'll do like:
```
import * as selectFeatureFlags from '../../../featureFlags/selectors';

...

selectFeatureFlags.enabled(state)
```

or in this case if you want to only pull in the one function maybe `selectEnabledFeatureFlags`